src/Mautic/Config.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Escopecz\MauticFormSubmit\Mautic;
+
+/**
+ * Configuration object for Mautic form submission. 
+ */
+class Config
+{
+    /**
+     * Curl verbose logging option
+     *
+     * @var bool
+     */
+    protected $curlVerbose = 1; 
```

Comment:
```suggestion
    protected $curlVerbose = true; 
```
Travis is complaining about this type mismatch.